src/components/common/Tile/PersonTile/PersonTile.jsx

```diff
@@ -0,0 +1,22 @@
+import { StyledPictureIcon } from "../SmallTile/styled";
+import { Image, ImageWrapper, Character, Name, Wrapper } from "./styled";
+
+export const PersonTile = ({ member }) => {
+  return (
+    <Wrapper>
+      <ImageWrapper>
+        {member.profile_path ? (
+          <Image
+            src={`https://image.tmdb.org/t/p/original${member.profile_path}`}
+            alt=""
+          />
+        ) : (
+          <StyledPictureIcon />
+        )}
+      </ImageWrapper>
+      <Character>
+        <Name>{member.name}</Name>
+      </Character>
```

Comment:
here is a <p> element inside another <p> element. We can't do this. please correct it.